src/lib/kw_db.sh

```diff
@@ -223,15 +222,132 @@ function select_from()
     return 22 # EINVAL
   fi
 
-  query="SELECT $columns FROM $table ;"
+  query="SELECT ${columns} FROM ${table} ;"
+
+  if [[ -n "${!_condition_array[*]}" ]]; then
+    where_clause="$(generate_where_clause '_condition_array')"
+    query="${query::-2} WHERE ${where_clause} ;"
+  fi
+
   if [[ -n "${order_by}" ]]; then
-    query="SELECT $columns FROM $table ORDER BY ${order_by} ;"
+    query="${query::-2} ORDER BY ${order_by} ;"
+  fi
+
+  cmd="sqlite3 -init "${KW_DB_DIR}/pre_cmd.sql" -cmd \"${pre_cmd}\" \"${db_path}\" -batch \"$query\""
+  cmd_manager "$flag" "$cmd"
+}
+
+# This function update the set of values in the table of given database
+# with the given conditions
+#
+# @table:     Table to select info from
+# @set:   An array reference of condition pairs that will be
+# updated in the db
+# @pre_cmd:   Pre command to execute
+# @_condition_array: An array reference of condition pairs
+# specifing the data that will be updated
+# @flag:      Flag to control function output
+# @db:        Name of the database file
+# @db_folder: Path to the folder that contains @db
+#
+# Return:
+# 2 if db doesn't exist; 22 if table is empty
+# 0 if succesful; non-zero otherwise
+function update_into()
```

Comment:
Make a dedicated commit for the implementation of this function (along with its doc and tests). It is indeed a necessary DB operation, but, for organizational reasons, we should separate it into an isolated commit.